gammapy/spectrum/tests/test_models.py

```diff
@@ -221,3 +230,55 @@ def test_pwl_index_2_error():
     eflux, eflux_err = pwl.energy_flux_error(1 * u.TeV, 10 * u.TeV)
     assert_quantity_allclose(eflux, 2.302585E-12 * u.Unit('TeV cm-2 s-1'))
     assert_quantity_allclose(eflux_err, 0.2302585E-12 * u.Unit('TeV cm-2 s-1'))
+
+
+@pytest.mark.xfail
+@requires_dependency('sherpa')
+@requires_data('gammapy-extra')
+def test_spectral_model_absorbed_by_ebl():
+
+    # Observation parameters
+    obs_param = ObservationParameters(alpha=0.2 * u.Unit(''),
+                                      livetime=5. * u.h,
+                                      emin=0.08 * u.TeV,
+                                      emax=12 * u.TeV)
+
+    # Target, PKS 2155-304 from 3FHL
+    name = 'test'
+    absorption = Absorption.read('$GAMMAPY_EXTRA/datasets/ebl/ebl_dominguez11.fits.gz')
+    pwl = PowerLaw(index=3. * u.Unit(''),
+                   amplitude=1.e-12 * u.Unit('1/(cm2 s TeV)'),
+                   reference=1. * u.TeV)
+
+    input_model = SpectralModelAbsorbedbyEbl(spectral_model=pwl,
+                                             absorption=absorption,
+                                             parameter=0.2)
+
+    target = Target(name=name, model=input_model)
+    
+    # Performance
+    filename = '$GAMMAPY_EXTRA/datasets/cta/perf_prod2/point_like_non_smoothed/South_5h.fits.gz'
+    cta_perf = CTAPerf.read(filename)
+
+    # Simulation
+    simu = CTAObservationSimulation.simulate_obs(perf=cta_perf,
+                                                 target=target,
+                                                 obs_param=obs_param)
+
+    # Model we want to fit
+    model = AbsorbedSpectralModel(spectral_model=PowerLaw(index=2.5 * u.Unit(''),
```

Comment:
Suggest to make the PowerLaw object on a line before and then pass it in here (less indented code)